### PR TITLE
New palette formats parsers

### PR DESCRIPTION
**Pull request by flabbet**
- Added png palette parsing
- Added Paint.net txt file parser
- Added Hex file parser
- Added Gimp GPL palette file parser
- Updated some Nuget packages
- #525

### CODE REVIEW

**Review by CPKreuz — CHANGES_REQUESTED** [review]
* When trying to drop a png file, it also opens the file open dialog
* When opening a big png file, maybe ask the user if they are sure they want to import a palette or are just trying to drop in a image
* I was not able to drop in any of GIMP's built in GPL file's. I guess they are using a old format? here's one of them: [Warm_Colors.txt](https://github.com/PixiEditor/PixiEditor/files/11302526/Warm_Colors.txt)
* I think it would be cool to be able to open .pixi files as palettes

**Comment by flabbet** [comment]
@CPKreuz I addressed issues you mentioned, good catches. But I ignored big png file

**Comment by flabbet** [comment]
Also I added discard current palette

**Review by CPKreuz — CHANGES_REQUESTED** [review]
Maybe disable these when there's no color in the palette

image

File: src/PixiEditor/Data/Localization/Languages/en.json [hunk 1]
```diff
@@ -426,6 +426,8 @@
     "BROWSE_PALETTES": "Browse palettes",
     "LOAD_PALETTE": "Load palette",
     "SAVE_PALETTE": "Save palette",
+    "DISCARD_PALETTE": "Discard palette",
+    "DISCARD_PALETTE_CONFIRMATION": "Are you sure you want to discard current palette? This cannot be undone.",
```
Comment (CPKreuz) [inline src/PixiEditor/Data/Localization/Languages/en.json:430]: I think this sounds better

Are you sure you want to discard *__the__* current palette? This cannot be undone.

**Review by CPKreuz — APPROVED** [review]
Looks good, maybe a small localization change tho

File: src/PixiEditor/Data/Localization/Languages/en.json (hunk 1 above)
Comment (flabbet) [reply src/PixiEditor/Data/Localization/Languages/en.json:430]: For me it sounds wierd, however this can be changed in POEditor

**Pull request merged**